src/plugins/dashboard/common/content_management/v2/types.ts

```diff
@@ -12,21 +12,18 @@ import type {
   SavedObjectCreateOptions,
   SavedObjectUpdateOptions,
 } from '@kbn/content-management-utils';
-import { ControlGroupSerializedState } from '@kbn/controls-plugin/common';
 import { DashboardContentType } from '../types';
-import { DashboardAttributes as DashboardAttributesV1 } from '../v1/types';
+import {
+  ControlGroupAttributesV1,
+  DashboardAttributes as DashboardAttributesV1,
+} from '../v1/types';
 
-type ControlGroupAttributesV2 = Pick<
-  ControlGroupSerializedState,
-  | 'panelsJSON'
-  | 'chainingSystem'
-  | 'controlStyle'
-  | 'ignoreParentSettingsJSON'
-  | 'showApplySelections'
->;
+export type ControlGroupAttributes = ControlGroupAttributesV1 & {
```

Comment:
Should this be called ControlGroupAttributesV2?

Reply:
The only place this type is used is in https://github.com/elastic/kibana/pull/193067/files#diff-a63df52e44eff3c16cf89fc96076e6d6782185b0074c113bbefe9611317d474eR40 where the import is renamed as such. But yeah, I can rename the type here if that is easier to understand.